apps/web/src/components/docs/Docs.tsx

```diff
@@ -141,9 +210,13 @@ export const Docs = ({ code = '', description = '', title = '', isLoading, child
               );
             },
             Frame: ({ className, ...props }: any) => {
+              const src = Array.isArray(props.children)
+                ? props.children.find((child) => child.type === 'img')?.props.src
+                : props.children.props?.src;
+
               return (
                 <div {...props}>
-                  <img alt="" src={`${MINTLIFY_IMAGE_URL}${props.children.props.src}`} />
+                  <img alt="" src={`${MINTLIFY_IMAGE_URL}${src}`} />
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** What happens here if src is undefined?‏